configs/parent.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://github.com/apache/trafficserver/tree/master/configs/parent.schema.json",
+  "title": "Traffic Server parents configuration",
+  "description": "parent configuration file structure. Licensed under Apache V2 https://www.apache.org/licenses/LICENSE-2.0",
+  "definitions": {
+    "destination": {
+      "type": "string",
+      "properties": {
+        "anyOf": [
+          { "dest_domain": { "type": "string" } },
+          { "dest_host": { "type": "string" } },
+          { "dest_ip": { "type": "string"} },
+          { "url_regex": { "type": "string"} }
+        ]
+      }
+    },
+    "internal": {
+      "type": "string",
+      "properties": {
+        "anyOf": [ "true", "false" ]
+      }
+    },
+    "max_simple_retries": {
+      "type": "number",
+      "properties": {
+        "anyOf": [ 1, 2, 3, 4, 5 ]
```

Comment:
For numbers, there are `minValue` and `maxValue` properties that are better for this.

Reply:
thanks @SolidWallOfCode, I've fixed the schema